docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_button.htm

```diff
@@ -24,6 +24,7 @@ <h1>Button</h1>
 >Use example</button>
 
 <p>The following table describes the supported button classes.</p>
+<img src="../../images/button.png" alt="photo of button element in Tizen">
```

Comment:
Duplicated img. Please remove this change.

Reply:
removed, also in 5.5 documentation